arvados/shell/package/install.sls

```diff
@@ -33,6 +33,12 @@ arvados-shell-package-install-pkg-installed:
       {%- endfor %}
 
 arvados-shell-package-install-gems-deps-pkg-installed:
+  {%- if grains.get('osfinger', '') == 'Oracle Linux Server-7' %}
+  cmd.run:
+    - name: "yum-config-manager --enable ol7_optional_latest"
+    - require_in:
+      - pkg: arvados-shell-package-install-gems-deps-pkg-installed
+  {%- endif %}
```

Comment:
This is the updated block I referred to [here](https://github.com/saltstack-formulas/arvados-formula/pull/14#discussion_r589104011).